circle.yml

```diff
@@ -1,9 +1,14 @@
 machine:
   python:
     version: 3.5.1
+  services:
+    - redis
 dependencies:
   pre:
     - pip install --upgrade pip
+    - pip install --upgrade virtualenv
+    - sudo add-apt-repository ppa:fkrull/deadsnakes-python2.7 -y
```

Comment:
@jbzdak We can't use Xenial on CircleCI because it only supports Ubuntu 12.04 and 14.04 images at this time. Good idea about the comment - I'll mention this won't be needed anymore after upgrade to 16.04.